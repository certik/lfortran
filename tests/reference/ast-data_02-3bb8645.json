{
    "basename": "ast-data_02-3bb8645",
    "cmd": "lfortran --indent --show-ast --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/data_02.f90",
    "infile_hash": "64c402dcb73aca8760e7059320fc541f9ff427be60bd8e9626cbee36",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "ast-data_02-3bb8645.stdout",
    "stdout_hash": "8c4c4392350b3ef5c4b6b96beb1955294c0835963e38b6fc93f514f5",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}