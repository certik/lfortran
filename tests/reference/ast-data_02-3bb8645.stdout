(TranslationUnit 
    [(Program 
        data2 
        () 
        [] 
        [(ImplicitNone 
            [] 
            ()
        )] 
        [(Declaration 
            (AttrType 
                TypeInteger 
                [] 
                () 
                None
            ) 
            [] 
            [(i 
            [] 
            [] 
            () 
            None 
            ())
            (iarx 
            [(1 
            3 
            DimensionExpr)
            (1 
            1 
            DimensionExpr)] 
            [] 
            () 
            None 
            ())
            (iary 
            [(1 
            3 
            DimensionExpr)
            (1 
            1 
            DimensionExpr)] 
            [] 
            () 
            None 
            ())] 
            ()
        )
        (Declaration 
            () 
            [(AttrData 
                [(DataImpliedDo 
                    [(FuncCallOrArray 
                        iary 
                        [] 
                        [(() 
                        i 
                        ())
                        (() 
                        1 
                        ())] 
                        [] 
                        []
                    )] 
                    () 
                    i 
                    1 
                    3 
                    ()
                )] 
                [1
                9
                1950]
            )] 
            [] 
            ()
        )] 
        [(Print 
            0 
            () 
            [(String 
                "1"
            )] 
            ()
        )
        (If 
            0 
            () 
            (Logical .true.) 
            [(Print 
                0 
                () 
                [(String 
                    "2"
                )] 
                ()
            )] 
            [(Print 
                0 
                () 
                [(String 
                    "3"
                )] 
                ()
            )] 
            () 
            () 
            ()
        )
        (Print 
            0 
            () 
            [(String 
                "4"
            )] 
            ()
        )
        (Print 
            0 
            () 
            [(String 
                "5"
            )] 
            ()
        )] 
        []
    )]
)
